src/Dialog/Dialog.js

```diff
@@ -16,6 +16,7 @@ var
 	Popup = require('../Popup'),
 	IconButton = require('../IconButton'),
 	BodyText = require('../BodyText'),
+	Divider = require('../Divider'),
```

Comment:
travis-ci build failed due to this line. are we assuming to use classes of divider?